pintracker/optracker/operationtracker.go

```diff
@@ -102,7 +103,14 @@ func (opt *OperationTracker) SetError(c cid.Cid, err error) {
 		return
 	}
 
-	if ph := op.Phase(); ph == PhaseDone || ph == PhaseError {
```

Comment:
Could we keep the phase if statement as it was, and move the put the following code above it:
```go
if ty := op.Type(); ty == OperationRemote {
    return
}

if ph := op.Phase(); ph == PhaseDone ...
```